cms/templates/course_outline.html

```diff
@@ -50,6 +51,45 @@ <h2 class="title title-3">${_("This course was created as a re-run. Some manual
     </div>
   </div>
   %endif
+
+  %if ora1_deprecation_info.get('ora1_components') or ora1_deprecation_info.get('ora1_enabled'):
+  <%
+    platform_name = microsite.get_value('platform_name', settings.PLATFORM_NAME)
+  %>
+  <div class="wrapper wrapper-alert wrapper-alert-error is-shown">
+    <div class="alert announcement">
+      <i class="feedback-symbol fa fa-warning"></i>
```

Comment:
In order to make this accessible we need to hide the actual icon from screen readers by adding an ```aria-hidden="true"```  on the ```<i>``` element and then adding some accessible text immediately afterwards like ```<span class="sr">${_("Warning")}</span>```